eclair-core/src/main/scala/fr/acinq/eclair/blockchain/OnChainWallet.scala

```diff
@@ -125,4 +125,30 @@ object OnChainWallet {
   final case class FundTransactionResponse(tx: Transaction, fee: Satoshi, changePosition: Option[Int]) {
     val amountIn: Satoshi = fee + tx.txOut.map(_.amount).sum
   }
+
+  final case class ProcessPsbtResponse(psbt: Psbt, complete: Boolean) {
+
+    import fr.acinq.bitcoin.psbt.UpdateFailure
+    import fr.acinq.bitcoin.scalacompat.KotlinUtils._
+
+    /** Transaction with all available witnesses. */
+    val partiallySignedTx: Transaction = {
+      var tx = psbt.global.tx
```

Comment:
this seems to be using a version of bitcoin-kmp that has not been released yet, for now we have to keep using `getGlobal` instead

Reply:
That's weird, how come it required it on my machine? I shouldn't have any override of the official `bitcoin-lib`, I'll look into it...

Reply:
I removed the previous last commit that made those changes (https://github.com/ACINQ/eclair/pull/2726/commits/7eb74a9c6cfbee31cb2b0d47fbc946f6cbc8c53b), I don't remember how I got that invalid 0.13 version in my local maven repository!